tech_soln/real_time_dashboards.rst

```diff
@@ -0,0 +1,422 @@
+.. _introduction:
+
+Real Time Dashboards
+#####################
+
+Citus provides real-time queries over large datasets. One workload we commonly see at
+Citus involves powering real-time dashboards of event data.
+
+For example, you could be a cloud services provider helping other businesses monitor their
+HTTP traffic. Every time one of your clients receives an HTTP request your service
+receives a log record. You want to ingest all those records and create an HTTP analytics
+dashboard which gives your clients insights such as the number HTTP errors their sites
+served. It's important that this data shows up with as little latency as possible so your
+clients can fix problems with their sites. It's also important for the dashboard to show
+graphs of historical trends.
+
+Alternatively, maybe you're building an advertising network and want to show clients
+clickthrough rates on their campaigns. In this example latency is also critical, raw data
+volume is also high, and both historical and live data are important.
+
+In this technical solution we'll demonstrate how to build part of the first example,
+but this architecture would work equally well for the second and many other use-cases.
+
+Running It Yourself
+-------------------
+
+There will be code snippets in this tutorial but they don't specify a complete system.
+There's `a github repo <https://github.com/citusdata/realtime-dashboards-resources>`_ with
+all the details in one place. If you've followed our installation instructions for running
+Citus on either a single or multiple machines you're ready to try it out.
+
+Data Model
+----------
+
+The data we're dealing with is an immutable stream of log data. We'll insert directly into
+Citus but it's also common for this data to first be routed through something like Kafka.
+Doing so has the usual advantages, and makes it easier to pre-aggregate the data once data
+volumes become unmanageably high.
+
+We'll use a simple schema for ingesting HTTP event data. This schema serves as an example
+to demonstrate the overall architecture; a real system might use additional columns.
+
+.. code-block:: sql
+
+  -- this is run on the master
+  CREATE TABLE http_request (
+    site_id INT,
```

Comment:
I think this should be zone_id, because we refer this field as zone_id in rest of the docs and the scripts in https://github.com/citusdata/technical-solution-resources.